__tests__/index.test.js

```diff
@@ -775,4 +776,28 @@ describe("Purchases", () => {
         });
       });
   });
+
+  describe("isConfigured", () => {
```

Comment:
Questions without a lot of context: isn't there any test we need to also add that iterates over all the appropriate methods and expects that we throw, if they are called before setup?

Reply:
that would be a good one. I guess I can store all functions we care about in an array and iterate over it?

Reply:
I added a test, I am not sure there's a better way of doing it but this works and will fail if a new function is added and it requires to check the instance [`6f3a8f0` (#310)](https://github.com/RevenueCat/react-native-purchases/pull/310/commits/6f3a8f04aa73c2d360b62cc57964b947b95ce2d7)